setup.py

```diff
@@ -0,0 +1,38 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="metrixpp",
+    version="0.0.1a",
+    author="Stefan Strobel",
+    author_email="stefan.strobel@shimatta.net",
+    description="Metrix++ is an extendable tool for code metrics collection and analysis.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/metrixplusplus/metrixplusplus",
+    packages=setuptools.find_packages(exclude=["metrixpp.tests"]),
+    package_data={
```

Comment:
is somebody adds a new folder (unlikely but still), how do we verify that the bundled package is still functioning?

Reply:
this is a good point.
I did not include the tests into the packet to reduce the size (and the config is quite bad if you want to include non *.py files)

It is though possible to run the tests from the repo against the package to verify nothing is missing...
I will try to add this to the action pipeline to create a local packet on every build and test against it.